### ISSUE TITLE
Use of parsets 

### ISSUE THREAD

**Issue by cyriltasse** [mhardcastle/ddf-pipeline#10]
I'm taking a look at the pipeline.py script. And I'm wondering if it wouldn't be better to use and invoque

`DDF.py TemplateLOFAR.parset --MSName = ...`

Instead of DDF.py plus a hundred parameters. Same for killMS.py calls, what do you think @mhardcastle ?

**Comment by mhardcastle** [comment]
Well yeah, we could do that. For DDF there are about 10 parameters that we never change, but well over twice that that need to be passed as arguments to the call, so it wouldn't make a big difference to the code; and it adds complexity in that the template parset needs to live in the ddf-pipeline source directory and be located by the code when it's needed. I guess you are thinking that this insulates us from possible future changes in the defaults?

**Comment by cyriltasse** [comment]
Yes, we could also call for the DDFacet parser, get a dict out of the default parset, change the dict depending on the step, save the dict back to disk and launch DDFacet. That would make the pipeine quite flexible I think. However I'm a bit lazy now, and I've started to just adapt the options in the current code.

**Comment by twshimwell** [comment]
Worth keeping this issue open?

**Comment by mhardcastle** [comment]
One day I would like to do what Cyril suggested -- the structure of the code is a bit clunky. But it's not urgent.
